Functional display

lparsons opened the pull request:
This PR is mostly work by @bdsinger to display the Functional connections data. I added some additional notes and resolved #8. There are certainly still some issues to resolve, but this appears to be functional enough to use as  preview (and thus perhaps ready to merge).

bdsinger committed:
First attempt at further incorporating functional connection type. Layout problems, but it draws.

bdsinger committed:
minor fixes based on npm run test

bdsinger committed:
some minor updates to cytoscape layout for functional connections and their line color

lparsons committed:
Fix removeNode function

The functional nodes are directional, like the chemical synapses, and
the removal code needs to check each direction (source and destination)
when removing a node.

Resolves #8

lparsons committed:
Update notes on functional data

Cleaned up some long lines in `tour-steps.js` and `tour.js`

hepcat72 reviewed (approved):
Nothing blocking.  There are a number of comments that I think should be tracked in GH issues, but overall, this is great work!

lparsons committed:
Add comments for clarification

lparsons committed:
Fix typos in tour

Co-authored-by: Robert Leach <rleach@princeton.edu>

lparsons committed:
Set default functional threshold to 2

hepcat72 committed:
Squeezed in the mock data in the bookmark dataset plot by making the markers smaller and adjusting the visualTime position.

Files checked in: src/client/scss/sidebars.scss src/server/populate-db/raw-data/datasets.json

hepcat72 committed:
Merge branch 'functional-display' of https://github.com/PrincetonUniversity/neuron-graph into functional-display

hepcat72 committed:
Included current protocol in creation of link to livereload.js.

Files checked in: src/client/js/init.js

hepcat72 committed:
Merge pull request #23 from PrincetonUniversity/use_current_protocol

Included current protocol in creation of link to livereload.js

hepcat72 committed:
Merge pull request #22 from PrincetonUniversity/dataset_marker_tweak

Dataset marker tweak

hepcat72 committed:
Somewhat of a guess on a fix for the timeout issue.

Files checked in: src/client/js/init.js
[skip ci]

hepcat72 committed:
Added the mock data as a default-displayed dataset.

Files checked in: src/client/js/controller/index.js src/client/js/init.js

hepcat72 committed:
Merge pull request #28 from PrincetonUniversity/display_default

Display default

bdsinger reviewed (approved):
sorry for delay!
